pkg/features/kube_features.go

```diff
@@ -837,6 +837,14 @@ const (
 	//
 	// Enable MinDomains in Pod Topology Spread.
 	MinDomainsInPodTopologySpread featuregate.Feature = "MinDomainsInPodTopologySpread"
+
+	// owner: @kerthcet
+	// kep: http://kep.k8s.io/3094
+	// alpha: v1.24
+	//
+	// Allow users to specify whether to take nodeAffinity/nodeTaint into consideration when
+	// calculating pod topology spread skew.
+	PodTopologySpreadNodePolicy featuregate.Feature = "PodTopologySpreadNodePolicy"
```

Comment:
I  am curious why you choose a different name style with `MinDomainsInPodTopologySpread`?
Why not use `NodePolicyInPodTopologySpread`?
It is important to have consistent naming of configurations with the same feature.

Reply:
Good cache, this is former designed in KEP and I was developing this feature the same time with `MinDomainsInPodTopologySpread`, but his feature merged first. I haven't notice the difference here yet. I will follow up this later.